src/debugger/server/SourceMapManager.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow strict-local */
+
+import type { SourceFile } from "./../../types.js";
+import { DebuggerError } from "./../common/DebuggerError.js";
+import invariant from "../common/invariant.js";
+
+export class SourceMapManager {
```

Comment:
This file deals with two source map formats. It's very important to clearly call out and distinguish between them. So I suggest you give a high level document about these two modes in class level doc comment here. So that people know there are two modes involving here.

Reply:
Also, it's good to call out these two scenarios are one for buck one for babel. And give two short path related examples.